dashboard/src/modules/components/QuisbyChartsComponent/index.jsx

```diff
@@ -0,0 +1,53 @@
+import "./index.less";
+
+import {
+  Divider,
+  Flex,
+  FlexItem,
+  Sidebar,
+  SidebarContent,
+  SidebarPanel,
+} from "@patternfly/react-core";
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import ChartGallery from "./ChartGallery";
+import PanelConent from "./PanelContent";
+import { getDatasets } from "actions/overviewActions";
+import { getQuisbyData } from "actions/quisbyChartActions";
+import { useNavigate } from "react-router-dom";
+
+const QuisbyChartsComponent = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const { datasets } = useSelector((state) => state.overview);
+
+  useEffect(() => {
+    if (Array.isArray(datasets) && datasets.length > 0) {
+      dispatch(getQuisbyData(datasets[0]));
+    } else {
+      dispatch(getDatasets());
+    }
+  }, [datasets, dispatch, navigate]);
```

Comment:
Is it really possible for this code to be invoked with `datasets` empty?  (Shouldn't we only be invoking this code if the user has selected one or more datasets?)

Reply:
UI has been updated and user can visit the comparison page without selecting any dataset from the overview page and selection can be made on the comparison page